rs/bitcoin/kyt/src/state.rs

```diff
@@ -128,52 +130,35 @@ impl Drop for FetchGuard {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(CandidType, Clone, Debug, Deserialize, Serialize)]
 pub struct Config {
-    pub network: Network,
+    pub network: BtcNetwork,
 }
 
 impl Default for Config {
```

Comment:
nit: I think this should not be necessary anymore (to instantiate the state one needs to go through init args)

Reply:
Yes, removed. Thanks